libs/community/langchain_community/llms/replicate.py

```diff
@@ -196,14 +189,21 @@ def _create_prediction(self, prompt: str, **kwargs: Any) -> Prediction:
                 "Please install it with `pip install replicate`."
             )
 
+        # get the replicate client
+        if self._replicate_client is None:
+            self._replicate_client = (
+                replicate_python.Client(api_token=self.replicate_api_token)
```

Comment:
This code is creating a client on each request. The client should be created once on init

Reply:
No, it is creating the client once per instance of the Replicate class. Doing it on init would make sense once this code is move to a separate package which can directly depend upon the replicate package. The code in its current home want to have a "soft" dependency on the replicate package which late imported.